internal/pkg/v2/infrastructure/redis/queries.go

```diff
@@ -35,7 +35,7 @@ func getObjectById(conn redis.Conn, id string, out interface{}) errors.EdgeX {
 func getObjectByHash(conn redis.Conn, hash string, field string, out interface{}) errors.EdgeX {
 	id, err := redis.String(conn.Do(HGET, hash, field))
 	if err == redis.ErrNil {
-		return errors.NewCommonEdgeX(errors.KindEntityDoesNotExist, fmt.Sprintf("%s doesn't exist in the database", field), err)
+		return errors.NewCommonEdgeX(errors.KindEntityDoesNotExist, fmt.Sprintf("fail to query object %T, %s doesn't exist in the database", out, field), err)
```

Comment:
("fail to query object %T, because %s: %s doesn't exist in the database", out, field, hash)

Reply:
Modified.